docs/howto/operator/gcp/natural_language.rst

```diff
@@ -68,7 +68,7 @@ Analyzing Entities
 Entity Analysis inspects the given text for known entities (proper nouns such as
 public figures, landmarks, etc.), and returns information about those entities.
 Entity analysis is performed with the
-:class:`~airflow.gcp.operators.natural_language.CloudLanguageAnalyzeEntitiesOperator` operator.
+:class:`~airflow.providers.google.cloud.operators.natural_language.CloudLanguageAnalyzeEntitiesOperator` operator.
 
 .. exampleinclude:: ../../../../airflow/gcp/example_dags/example_natural_language.py
```

Comment:
I think the example path will also require change to new import ✅

Reply:
should the tests also move in the same way?

Reply:
Yes, tests should also be moved accordingly